e2e/framework/README.md

```diff
@@ -0,0 +1 @@
+# rook-test-framework
```

Comment:
can we get instructions on running the tests? maybe instead of this readme, it could go up in the `e2e` folder